packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -115,8 +120,14 @@ const NewExperimentForm: FC<NewExperimentFormProps> = ({
       targetURLRegex: initialValue?.targetURLRegex || "",
       description: initialValue?.description || "",
       guardrails: initialValue?.guardrails || [],
-      variations:
-        initialValue?.variations || getDefaultVariations(initialNumVariations),
+      variations: initialValue?.variations
+        ? initialValue.variations.map((variation) => {
+            return {
+              ...variation,
+              id: generateVariationId(),
```

Comment:
This looks like it would override the `id` if it exists, do we want that to happen? If not, perhaps reorder it:

```suggestion
              id: generateVariationId(),
              ...variation
```

Reply:
Currently, it shouldn't matter as the id's that are generated are temporary, and get regenerated each time a user visits a particular experiment. I've committed the suggestion, though, in the event that we do ever persist these ids. Thanks!